core/services/subscription.go

```diff
@@ -248,16 +248,16 @@ func (sub ManagedSubscription) listenToLogs(q ethereum.FilterQuery) {
 	}
 }
 
-// Manually retrieve old logs since SubscribeToLogs(logs, filter) only returns newly
+// Manually retrieve old logs since SubscribeFilterLogs(ctx, filter, chLogs) only returns newly
 // imported blocks: https://github.com/ethereum/go-ethereum/wiki/RPC-PUB-SUB#logs
-// Therefore TxManager.GetLogs does a one time retrieval of old logs.
+// Therefore TxManager.FilterLogs does a one time retrieval of old logs.
 func (sub ManagedSubscription) backfillLogs(q ethereum.FilterQuery) map[string]bool {
 	backfilledSet := map[string]bool{}
 	if q.FromBlock == nil {
 		return backfilledSet
 	}
 
-	logs, err := sub.logSubscriber.GetLogs(q)
+	logs, err := sub.logSubscriber.FilterLogs(context.TODO(), q)
```

Comment:
```suggestion
	logs, err := sub.logSubscriber.FilterLogs(context.Background(), q)
```

Any reason for the TODO?